store/LocalStore.js

```diff
@@ -3,37 +3,27 @@
 const lru = require('tiny-lru')
 
 function LocalStore (timeWindow, cache, app, continueExceeding) {
-  this.lru = lru(cache || 5000)
-  this.interval = setInterval(beat.bind(this), timeWindow).unref()
+  this.lru = lru(cache || 5000, timeWindow)
   this.app = app
   this.timeWindow = timeWindow
   this.continueExceeding = continueExceeding
-
-  app.addHook('onClose', (instance, done) => {
-    clearInterval(this.interval)
-    done()
-  })
-
-  function beat () {
-    this.lru.clear()
-    this.msLastBeat = null
-  }
 }
 
 LocalStore.prototype.incr = function (ip, cb) {
-  let current = this.lru.get(ip) || 0
-  this.lru.set(ip, ++current)
+  const current = this.lru.get(ip) || { count: 0, iterationStartMs: Date.now() }
+  const isNewItem = current.count === 0
+
+  current.count++
+
+  // We want to update the TTL only if it's a new item (because it doesn't have any TTL)
+  // Passing false to the 3rd parameter = update the TTL AND I don't know if the item with that key already exist
+  // Passing true means don't update the TTL AND I'm sure there is an item with that key
+  this.lru.set(ip, current, !isNewItem)
```

Comment:
We should also update the TTL when `continueExceeding` is `true` AND the user has already reached his rate limit, right?

I'm keeping the bug as it's the same bug as #207 because:
1. #215 bug is more important IMO
2. this is not the PR for that, even though I'm changing the related code we should think about how to approach this as the store doesn't know when the client reached its maximum request in the time window...